src/host/messaging.ts

```diff
@@ -18,7 +18,7 @@ export function initializeMessaging() {
         // See: https://blog.mattbierner.com/vscode-webview-web-learnings/

```

Comment:
You were saying that we should ignore this comment.  Should we remove it if it doesn't apply anymore?